Foundation/NSGeometry.swift

```diff
@@ -36,7 +37,7 @@ public struct CGFloat {
 extension CGFloat: Comparable { }
 
 public func ==(lhs: CGFloat, rhs: CGFloat) -> Bool {
-    return lhs.native == rhs.native
+    return abs(lhs.native - rhs.native) < DBL_EPSILON
```

Comment:
you're right:
https://github.com/apple/swift/blob/master/stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb#L226

The problem then is that `NSScanner` reads `0.6` `Double` as `0.600000000000000004` or 
`0.599999999999999996`, which is essentially the same number. Shall I fix the test to compare rects, points and sizes with a certain precision?